src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const calcButtons = ['AC', '+/-', '%', '/', '7', '8', '9', 'X', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+
+function ShowButtons() {
+  function CreateElement(text, index) {
+    let className = '';
+    if ((index + 1) % 4 === 0 || (index + 1) === 19) {
+      className = ' makeMe-orange';
+    } else if ({ text }.text === '0') {
+      className = ' calc-button-0';
+    }
+
+    return (
+
+      <button key={text} className={className} type="button">{text}</button>
+
+    );
+  }
+
+  return (
+    calcButtons.map(CreateElement)
+  );
+}
+
+export default ShowButtons;
```

Comment:
Great job creating the calculator design. However, per the project requirements, the `Calculator.js` component should have at least one child component. I would recommend creating a new component for the `CreateElements` method in a separate file and then using that component here passing the props into `CreateElements`. That should get the job done.

- Optional
As per the React conventions, the name of the component is the same as the name of the file. I would recommend you rename the `ShowButtons` component to `Calculator`.